iointel/src/agent_methods/tools/agno/api.py

```diff
@@ -18,4 +36,11 @@ def make_request(
         headers: Optional[Dict[str, str]] = None,
         json_data: Optional[Dict[str, Any]] = None,
     ) -> str:
-        return self._tool.make_request()
+        return self._tool.make_request(
+            endpoint=self.endpoint,
+            method=self.method,
+            params=self.params,
+            data=self.data,
+            headers=self.headers,
+            json_data=self.json_data,
```

Comment:
```suggestion
            endpoint=endpoint,
            method=method,
            params=params,
            data=data,
            headers=headers,
            json_data=json_data,
```
this should be passing parameters from the tool call, not from `self`; it's `_tool` job to decide if it wants to default to smth